packages/tizen_notification/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 1.0.0
```

Comment:
There are some argument options that are not included in the current signature because notification api doesnt behave the same across different devices, I need to do some testing before I add more options. I'll release it as 0.1.0.